src/tslint-cli.ts

```diff
@@ -207,6 +207,22 @@ const processFile = (file: string) => {
         process.exit(1);
     }
 
+    const buffer = new Buffer(256);
+    buffer.fill(0);
+    const fd = fs.openSync(file, "r");
+    try {
+        fs.readSync(fd, buffer, 0, 256, null);
+        if (buffer.readInt8(0) === 0x47 && buffer.readInt8(188) === 0x47) {
+            // MPEG transport streams use the '.ts' file extension. They use 0x47 as the frame
+            // separator, repeating every 188 bytes. It is unlikely to find that pattern in
```

Comment:
Just for fun, I grabbed about the top 1000 TypeScript repos off Github which combined had about 65,000 `.ts` files. About 700, or 1%, had a "G" as the first and the 189th character.

This seemed surprisingly high, given that "/" and "i" are by far (and expectedly) the most common first characters.

Turns out every one of the files with two "G"s was an MPEG transport stream file. 😉